nnvm/tests/python/frontend/onnx/test_forward.py

```diff
@@ -189,33 +189,62 @@ def test_unsqueeze():
 
     np.testing.assert_allclose(out_shape, tvm_out.shape)
 
-def verify_gather(in_shape, indices, axis=0):
-    indices_src = np.array(indices, dtype="int32")
+def verify_gather(in_shape, indices, axis, dtype):
+    x = np.random.uniform(size=in_shape).astype(dtype)
+    indices = np.array(indices, dtype="int32")
+    out_np = np.take(x, indices, axis=axis)
 
-    x = np.random.uniform(size=in_shape)
-    out_np = np.take(x, indices_src, axis=axis)
-
-    y = helper.make_node("Gather", ['in'], ['out'], indices=indices, axis=axis)
+    y = helper.make_node("Gather", ['in', 'indices'], ['out'], axis=axis)
 
     graph = helper.make_graph([y],
                               'gather_test',
                               inputs = [helper.make_tensor_value_info("in",
-                                  TensorProto.FLOAT, list(in_shape))],
+                                      TensorProto.FLOAT, list(in_shape)),
+                                  helper.make_tensor_value_info("indices",
+                                      TensorProto.INT32, list(indices.shape))],
                               outputs = [helper.make_tensor_value_info("out",
-                                  TensorProto.FLOAT, list(out_np.shape))])
-
+                                      TensorProto.FLOAT, list(out_np.shape))])
     model = helper.make_model(graph, producer_name='gather_test')
 
+    def _get_tvm_output(model, x, indices, target, ctx, out_shape):
```

Comment:
Suggest to extend existing helper function to receive multiple args instead of defining new.
A reference implementation can be found in tensorflow test cases.